src/app/services/data/information/information.ts

```diff
@@ -1,6 +1,19 @@
+export const enum Importance {
+  Low = 'low',
+  High = 'high'
+}
+
+export const enum InformationType {
+  Device = 'device',
+  Location = 'location',
+  Other = 'other'
+}
+
 export interface Information {
   readonly proofHash: string;
   readonly provider: string;
   readonly name: string;
   readonly value: string;
+  readonly important: Importance;
```

Comment:
I think the `important` property should be renamed to `importance` for consistency.